config.py

```diff
@@ -22,10 +22,31 @@ def next_year():
         config = Configuration(year=new_year)
         db.session.add(config)
         db.session.commit()
-        Configuration.update_current_year(config.id)
 
     except Exception as e:
         db.session.rollback()
         raise e
 
     return redirect(url_for("index"))
+
+# This method allows the admin to move on to the next academic year for all users.
+@config_bp.route('/next_year', methods=['POST'])
+@login_required
+@check_access_level(Role.ADMIN)
+def next_year():
+    current_year = Configuration.query.filter_by(is_current_year=True).first()
+    try:
+        is_new_year = Configuration.query.filter_by(year=current_year.year + 1).first()
+        # Check if the following year already exists
+        if is_new_year is None:
+            return make_response("Please create a new year before moving on to the next one", 500)
```

Comment:
The entry can probably be inserted automatically if needed to improve user friendlyness. Besides, `make_response` is not a very elegant way to return feedback to the user.